librz/core/cmd/cmd_eval.c

```diff
@@ -68,16 +83,56 @@ RZ_API bool rz_core_theme_load(RzCore *core, const char *name) {
 	char *system_themes = rz_path_system(RZ_THEMES);
 	char *home_file = rz_file_path_join(home_themes, name);
 	char *system_file = rz_file_path_join(system_themes, name);
+
+
 	free(system_themes);
 	free(home_themes);
 
-	if (!load_theme(core, home_file)) {
+
+	char *system_file_cpy = strdup(system_file);
+
+
+	if(name[len-1] == '?' && (question_mark_count == 1)){	
+		system_file_cpy[strlen(system_file_cpy) - 1] = '\0';
+		FILE *path = rz_sys_fopen(system_file_cpy,"r");
+
+		if(!path){
+			RZ_LOG_ERROR("Error in opening colourscheme file at '%s'\n", system_file_cpy);
+			failed = true;
+			return !failed;
+		}
+		else{
+		rz_cons_printf("%s","Color theme details!\n");
+		}
+
+		bool beginflag = false;		
+		char line[256];
+		while((fgets(line , sizeof(line) , path) != NULL) && (strcmp(line,"# END\n") != 0) ){
+			
+			if(beginflag == true){
+
+				rz_cons_println(line);
```

Comment:
Instead of just printing, you need to create the structure and store all that metainformation somewhere. Another option would be to use key-value string storage, e.g. SDB: `librz/util/sdb/sdb.h`